### ISSUE TITLE
Spell glyphs not generating in player inventory with several mods enabled

### ISSUE THREAD

**Issue by TNTftw21** [Kalciphoz/kRPG#40]
Via @JakieJake9725 on Discord:

With several mods active that add items to the hotbar (in this case, Calamity, Joost's Mod, and AlchemistNPC), there isn't enough room to add kRPG's items, and so some items (typically the spell glyphs) end up not being generated.

This is probably an issue with adding items to newly generated players in general - tModLoader may just increment a counter representing the inventory slot it's on and doesn't do any kind of error checking to add items directly to the inventory instead of running off the edge of the hotbar.

**Comment by FairfieldTekLLC** [comment]
Is this when a new player joins and it suppose to give you stuff for being a new player?

**Comment by TNTftw21** [comment]
Yes, seems like. I don't know if that's specifically done when a new character is generated, or when they join a world for the first time.
